README.md

```diff
@@ -119,6 +119,10 @@ aws:
     attribute_names:
       - ApproximateReceiveCount
       - SentTimestamp
+sqs:
+  log_level: debug
+sns:
+  log_level: info
```

Comment:
@atyndall the default [`logger`](https://docs.aws.amazon.com/AWSRubySDK/latest/AWS/Core/Configuration.html#logger-instance_method) is `nil`, so even if you change the level to info, it won't output `info.log`. If you are seeing the skd log in a file, it's probably because you setting it in somewhere else `AWS.config(logger: ...)`.

I might be missing something in here, but to have the sdk configured to log to a file or `stdout`, we also need to configure `logger: ...`, otherwise it won't output.